florist/api/routes/server/training.py

```diff
@@ -66,38 +39,46 @@ def start(
             {"error": <error message>}
     """
     try:
-        # Parse input data
-        if model not in Model.list():
-            error_msg = f"Model '{model}' not supported. Supported models: {Model.list()}"
-            return JSONResponse(content={"error": error_msg}, status_code=400)
-
-        model_class = Model.class_for_model(Model[model])
-        clients_info_list = ClientInfo.parse(clients_info)
+        job_collection = request.app.database[JOB_COLLECTION_NAME]
+        result = await job_collection.find_one({"_id": job_id})
+        job = Job(**result)
```

Comment:
Maybe I am missing something, but should we check the job status is not started and throw an error if not

Reply:
A related question is how do we monitor the job and update the status.  Not sure if we should set to in progress after launching or if another ticket we will build out the functionality to monitor the job and update the status from in progress to finished

Reply:
This is what I'm working on right now. I'm using Redis' publisher-subscriber mechanism to notify a listener that is going to run in the background and update the job in the database when there is a change of status. It's working really well right now, I'm just figuring out the unit tests.

The check for not started and the change to in progress is also being done on my upcoming PR but I can move it here if you think it's necessary, I think it's a better place anyway.

Reply:
Amazing, excited to see that PR, it sounds awesome. No worries about moving it here if its already in the other one.